sw/airborne/subsystems/ins/ins_int.c

```diff
@@ -374,6 +378,7 @@ void ins_int_update_gps(struct GpsState *gps_s)
 
 #if INS_USE_GPS_ALT
   vff_update_z_conf(((float)gps_pos_cm_ned.z) / 100.0, INS_VFF_R_GPS);
+  vff_update_vz_conf(((float)gps_speed_cm_s_ned.z) / 100.0, INS_VFF_VZ_R_GPS);
```

Comment:
that would make sense since since velocity or altitude could come from different sensors or at different speed after all